rfc-000-template.md

```diff
@@ -16,6 +16,7 @@ language to help clarify your intentions.
 
 ## Next steps
 
-Describe any changes to code or documentation that will need to happen if this
-RFC is accepted. For example, this might be modifying the playbook or a template
-project.
+- [ ] Describe any changes to code or documentation that will need to happen if
```

Comment:
Should we recommend that any tasks are opened as issues and then referenced? It might make them easier to keep track of.

Reply:
I wondered about that, but the issues wouldn't get created until the RFC was accepted, so I wasn't sure how to recommend doing it :)

Reply:
Yeah good point. Maybe agree the RFC, then make the issues and add a references before merging? Or we could just have issues without references?

Reply:
I've added another sentence to say that people should come back and add links if appropriate. That's probably enough?